tensorflow/python/autograph/converters/control_flow.py

```diff
@@ -156,19 +156,31 @@ def _create_nonlocal_declarations(self, loop_vars):
 
   def _create_state_functions(
       self, loop_vars, nonlocal_declarations, getter_name, setter_name):
-    template = """
-      def getter_name():
-        return state_vars,
-      def setter_name(loop_vars):
-        nonlocal_declarations
-        state_vars, = loop_vars
-    """
-    return templates.replace(
+    if loop_vars:
+      template = """
+        def getter_name():
+          return state_vars,
+        def setter_name(loop_vars):
+          nonlocal_declarations
+          state_vars, = loop_vars
+      """
+      return templates.replace(
         template,
         nonlocal_declarations=nonlocal_declarations,
         getter_name=getter_name,
         setter_name=setter_name,
         state_vars=tuple(loop_vars))
+    else:
+      template = """
+        def getter_name():
+          return ()
+        def setter_name(loop_vars):
+          pass
+      """
+      return templates.replace(
+        template,
```

Comment:
Looks like the linter complains about the indentation here - needs to be 4 spaces.

Reply:
@mdanatg , I pushed out a new commit which the linter will hopefully take a liking to :)

thanks